 src/luajit.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/luajit.c b/src/luajit.c
index 73e29d44..1c04bac2 100644
--- a/src/luajit.c
+++ b/src/luajit.c
@@ -584,3 +584,5 @@ int main(int argc, char **argv)
   return (status || smain.status > 0) ? EXIT_FAILURE : EXIT_SUCCESS;
 }
 
+int __stdcall WinMain(void* hInstance, void* hPrevInstance, char* pCmdLine, int nShowCmd)
+{ return main(__argc, __argv); }
